homeassistant/components/fronius/coordinator.py

```diff
@@ -55,7 +59,13 @@ async def _async_update_data(self) -> dict[SolarNetId, Any]:
         async with self.solar_net.coordinator_lock:
             try:
                 data = await self._update_method()
+                if self._failed_update_count != 0:
```

Comment:
Please only wrap the line that can raise in the try... except block. Use and `else:` block if needed.